docs/en/03-benchmark/benchmark.md

```diff
@@ -326,7 +326,7 @@ Users can directly test the speed through [model profiling](../02-how-to-run/pro
     <td align="center" colspan="1">fp32</td>
   </tr>
     <tr>
-    <td align="center"><a href="https://github.com/open-mmlab/mmocr/tree/master/configs/textdet/dbnet/dbnet_r18_fpnc_1200e_icdar2015.py">DBNet</a></td>
+    <td align="center"><a href="https://github.com/open-mmlab/mmocr/blob/1.x/configs/textdet/dbnet/dbnet_resnet18_fpnc_1200e_icdar2015.py">DBNet</a></td>
```

Comment:
pls. update mmocr related links in `benchmark_edge.md`, `quantization.md` as well.